src/testup/ui/preferences.rb

```diff
@@ -94,7 +94,7 @@ def event_save_config(config)
       Log.info "event_save_config(...)"
       Log.debug config.inspect
       # Save editor config:
-      application = config['editor']['application']
+      application = config['editor']['executable']
```

Comment:
Oh - isn't this the core of this but? Using the wrong key? Was the `onkey` events `preferences.ts` necessary?

Reply:
Yes, 'application' is a wrong key.